ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -219,16 +219,15 @@ message DisclosedContract {
   // Required
   string contract_id = 2;
 
-  // The contract arguments
+  // The contract arguments as typed Record
   // Required
```

Comment:
This one can be optional no?

Reply:
Also we should probably not reuse the proto indexes

Reply:
> Also we should probably not reuse the proto indexes

As discussed offline, probably not an issue if it doesn't break wire backwards compatibility

Reply:
Yes, apparently this works nicely in proto

- new server can deal with old client which will only populate one of the fields
- old server will pick the later field (blob=5) when confronted with both

Reply:
I have changed this now to be backwards compatible Apparently we have enough downstream components to make this a necessity